applications/EdgeSnap.md

```diff
@@ -0,0 +1,199 @@
+
+The evolution from product to product as a service is the end goal for EdgeSnap. After milestone 1, we will start adding and/or will be keen to assist different [Ecosystem dApps](https://polkadot.network/ecosystem/dapps/) to add Snap support.
+
+### **Overview**
+
+- **Total Estimated Duration:** 3 Months
+- **Full-Time Equivalent (FTE):** 2.4
+- **Total Costs:** 120,000 USDT
+- **Upfront** : 18,000 USDT
+
+### **Milestone 1 | Complete Back-end Flow**
+
+- **Estimated duration:** 1 Month
+- **FTE:** 2.4
+- **Costs:** 45,000 USDT
+
+This implementation will showcase the working dynamics and final UI for a limited number of chains, providing users with a complete overview of the anticipated end product. We will also evaluate the UX flow and make iterative improvements based on the feedback received through different channels.
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| --- | --- | --- |
+| **0b.** | Documentation | Publish detailed tech as well as non-tech docs along with beginner-friendly video walkthroughs. |
+| **0c.** | Article | Publish 2 **articles, one** explaining the EdgeSnap in a nutshell and the other one explaining why we decided to evolve the EdgeSnap to the broader substrate ecosystem. We will also host a weekly public Cohort program inviting the maintainers of different Polkadot ecosystem dApps to add Snap compatibility. |
+| 1. | Custom npm packages | Develop substrate-snaps, metamask-substrate-adapter and metamask-substrate-types based on current Polkadot&Kusama-only implementation by ChainSafe. |
+| 2. | Compilation flow changes | Add support to compile individual chains' snap apps alongside the default Universal Snap app. The Snap Store is already ready to link the precompiled individual Snap apps to corresponding chains. While in the current MVP all Install buttons are linked to the Universal Snap app temporarily. |
+| 3. | UI changes | Add an RPC switcher element corresponding to each chain under Snap Store. This will need the individual Snap apps to be precompiled in parallel for all the added RPCs. |
+| 4. | Feedback collection & analysis | Collect feedback from different devs and users to improve the UI/UX flow further. |
+
+###
+
+### **Milestone 2 | First batch Integration**
+
+- **Estimated Duration:** 1 month
+- **FTE:** 1.8
+- **Costs:** 28,500 USDT
+
+In milestone 2, we are looking to achieve seamless integration of approximately half of the total Substrate chains.
+
+| Number | Deliverable | Specification |
+| --- | --- | --- |
+| **0a.** | License | Apache 2.0 / GPLv3 / MIT / Unlicense |
+| --- | --- | --- |
+| **0b.** | UpdateDocumentation | Update the docs with newly added chains' info. |
+| 1. | Chains' Integration | Do integration for approx half (45-55) of the total chains. {Only the relaychains, parachains and solochains available on polkadot-js will be integrated.} |
```

Comment:
Does this deliverable mean that you are going to create the snaps for all the (45-55) chains yourself? Integrating sounds a lot like you will have to rely on third parties to be able to deliver this milestone.

Reply:
Yes! We do not rely on any third parties and all the (tech/ non-tech) deliverables will be delivered by us.